docs/docusaurus.config.js

```diff
@@ -5,6 +5,7 @@ const config = {
   title: "Tobira documentation",
   url: "https://elan-ev.github.io/",
   baseUrl: "/tobira/",
+  trailingSlash: false, // This is better when deploying via GH pages.
```

Comment:
Why

Reply:
Here, scroll down a bit from the anchor:

https://docusaurus.io/docs/deployment#docusaurusconfigjs-settings

Reply:
Can we link that in the comment? :thinking: